package.json

```diff
@@ -31,21 +31,18 @@
   "homepage": "https://github.com/NodeSecure/github#readme",
   "dependencies": {
     "@myunisoft/httpie": "^1.10.0",
-    "tar-fs": "^2.1.1"
+    "tar-fs": "^2.1.1",
+    "undici": "^5.28.2"
```

Comment:
```suggestion
```

Please use `@myunisoft/httpie` ^^

`@myunisoft/httpie` exports everything that concerns `MockAgent`:

For instance:
```js
import { MockAgent, getGlobalDispatcher, setGlobalDispatcher } from "@myunisoft/httpie";
```

Reply:
ok